docs/devel/release-process.md

```diff
@@ -62,6 +62,13 @@ $ git push origin v0.x.0
 
 ## Post release tasks
 
+- Create a branch for this release named `v0.x-stable`, this will be used as stable branch where fixes would be backported:
+
+```bash
+$ git checkout -b v0.x-stable
```

Comment:
I'd suggest to use release instead of stable as it's more clear:

```suggestion
$ git checkout -b v0.x-release
```

As done by Cilium

or `release-0.x` as done by Kubernetes